utils/testmap.py

```
@@ -0,0 +1,142 @@
+import folium
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)